src/workingtitle-aircraft-tbm930/SimObjects/Airplanes/Asobo_TBM930/model/TBM930_interior.xml

```diff
@@ -0,0 +1,2228 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<ModelInfo>
+	<LODS>
+		<!-- Highest LOD -->
+		<!-- minimum display size in % (should be bounding sphere vertical screen size, sphere will often be slightly larger than you expect) -->
+		<!-- default minSize is 0 -->
+		<LOD minSize="100" ModelFile="TBM930_interior_LOD00.gltf"/>
+		<LOD minSize="70" ModelFile="TBM930_interior_LOD01.gltf"/>
+		<LOD minSize="20" ModelFile="TBM930_interior_LOD02.gltf"/>
+		<LOD minSize="10" ModelFile="TBM930_interior_LOD03.gltf"/>
+		<LOD minSize="5" ModelFile="TBM930_interior_LOD04.gltf"/>
+		<LOD minSize="2" ModelFile="TBM930_interior_LOD05.gltf"/>
+	</LODS>
+	<Behaviors>
+		<!-- Includes ###################################### -->
+		<Include ModelBehaviorFile="Asobo\Common.xml"/>
+		<Include ModelBehaviorFile="Asobo\GlassCockpit\AS3000_enhanced.xml"/>
```

Comment:
I don't think these are going to work any more, those files don't exist in this repo.

Reply:
Oh yeah, missed that one. Easy fix to just change the file path.

Reply:
Fixed. af150ce11262a3bb1dec23a07298f3a7aaa56562